crowdstrike/table_crowdstrike_user_role.go

```diff
@@ -0,0 +1,101 @@
+package crowdstrike
+
+import (
+	"context"
+
+	"github.com/crowdstrike/gofalcon/falcon"
+	"github.com/crowdstrike/gofalcon/falcon/client"
+	"github.com/crowdstrike/gofalcon/falcon/client/user_management"
+	"github.com/crowdstrike/gofalcon/falcon/models"
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+func tableCrowdStrikeUserRole(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "crowdstrike_user_role",
+		Description: "User roles define the level of access of a user in Falcon.",
+		List: &plugin.ListConfig{
+			Hydrate: listCrowdStrikeUserRole,
+		},
+		Get: &plugin.GetConfig{
+			Hydrate:    getCrowdStrikeUserRole,
+			KeyColumns: plugin.AllColumns([]string{"id"}),
+		},
+		Columns: []*plugin.Column{
+			{Name: "id", Description: "The ID of the role.", Type: proto.ColumnType_STRING},
```

Comment:
Is there any other information in the role, like permissions, creation times, etc.? This table currently feels very light and I'm not sure how useful it is